packages/react-components/src/components/AppFrame/AppFrame.stories.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from 'react';
+
+import * as Icons from '@livechat/design-system-icons';
+
+import { AppFrameProvider } from '../../providers/AppFrameProvider';
+import { Avatar } from '../Avatar';
+import { Icon } from '../Icon';
+import { Tooltip } from '../Tooltip';
+
+import { AppFrame } from './AppFrame';
+import { NavBarItem, NavBarList } from './components';
+import {
+  ExampleAppContent,
+  getArchivesSubMenu,
+  getChatsMenu,
+  getEngageSubMenu,
+} from './stories-helpers';
+
+const defaultImage =
+  'https://cdn-labs.livechat-files.com/api/file/lc/img/100019504/df59da4b5b0cdb6030efb08787fd255d.jpg';
+
+export default {
+  title: 'Components/AppFrame',
+  component: AppFrame,
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+const SectionsWithToggle = ['chats', 'engage', 'archives'];
+
+export const Default = (): React.ReactElement => {
+  const [activeItem, setActiveItem] = React.useState('home');
+  const [activeSubItem, setActiveSubItem] = React.useState(0);
+  const [topBarVisible, setTopBarVisible] = React.useState(true);
+
+  const getSubNav = () => {
+    switch (activeItem) {
+      case 'chats':
+        return getChatsMenu(activeSubItem, setActiveSubItem);
+      case 'engage':
+        return getEngageSubMenu(activeSubItem, setActiveSubItem);
+      case 'archives':
+        return getArchivesSubMenu(activeSubItem, setActiveSubItem);
+      default:
+        return null;
+    }
+  };
+
+  return (
+    <AppFrameProvider>
+      <AppFrame
```

Comment:
I'd suggest using the AppFrameProvider in AppFrame internally, but to leave it exported externally as well -> thanks to that, developers wouldn't need to declare it explicitly but will have the ability to use the sidenav controls if needed separately